packages/webapp/pages/onboarding.tsx

```diff
@@ -224,6 +230,15 @@ export function OnboardPage(): ReactElement {
       return setActiveScreen(OnboardingStep.Sources);
     }
 
+    if (
+      (showOnboardingSources && activeScreen === OnboardingStep.Sources) ||
```

Comment:
Is this not going to be behind any existing experiment we have for plus subscription?

Reply:
sources is a running experiment, you can't rely on it yet.

Reply:
It should actually be it's own evaluation experiment